toolchain/check/inst.cpp

```diff
@@ -61,6 +65,28 @@ auto AddInstInNoBlock(Context& context, SemIR::LocIdAndInst loc_id_and_inst)
   return inst_id;
 }
 
+auto AddDependentActionInst(Context& context,
```

Comment:
Clarify in a comment that this is about template dependence? as opposed what is being handled in `FinishInst`?

Reply:
Where do you want this comment? The comment in the header mentions this: "The instruction is expected to be a dependent template instantiation action."